XLS-47d-PriceOracles/README.md

```diff
@@ -0,0 +1,468 @@
+<pre> 
+   Title:        <b>Oracles on XRP Ledger</b>
+   Revision:     <b>3</b> (2023-10-18)
+ <hr>  Author: <a href="mailto:gtsipenyuk@ripple.com">Gregory Tsipenyuk</a>
+   Affiliation:  <a href="https://ripple.com">Ripple</a>
+ 
+ </pre>
+ 
+ # Price Oracles on XRP Ledger
+ 
+ ## Abstract
+
+ This proposal adds an on-chain `PriceOracle` object to the XRPL ledger. A blockchain oracle is a system or service that acts as a bridge between a blockchain network and the external world, providing off-chain data or information to decentralized applications (dApps) on the blockchain. Oracles are used to bring real-world data, for instance  market prices, exchange rates, interest rates, or weather conditions onto the blockchain, enabling dApps to access and utilize information that resides outside the blockchain. This document outlines a new protocol for price oracles the on XRPL ledger, and provides guidelines for developers and system architects to implement and utilize this solution effectively. This proposal introduces a new on-ledger `PriceOracle` object and the transactions to create, delete, and update the `PriceOracle`. It also adds the `get_aggregate_price` API, to retrieve an `aggregate mean`, `trimmed mean`, and `median` for the provided price oracles. This feature requires an amendment.
```

Comment:
Just clarifying -- is the intention that this spec should support non-price data like Weather data, or is that just an example of a more general oracle?

Reply:
The intention is to support Price Data only.

Reply:
Awesome, thanks for clarifying and updating the title.